rudder-agent/SOURCES/Makefile

```diff
@@ -166,6 +166,10 @@ ifneq (true,$(NO_LDSO))
 	$(INSTALL) -m 644 rudder.conf $(DESTDIR)/etc/ld.so.conf.d/
 endif
 
+        # Install /etc/bash_completion.d/rudder in order to autocomplete rudder commands
+	mkdir -p $(DESTDIR)/etc/bash_completion.d
+	$(INSTALL) -m 644 rudder.conf $(DESTDIR)/etc/bash_completion.d/
```

Comment:
You sure this is the right file ??? rudder.conf is also installed in ld.so.conf...